lib/manager/components/HomeProjectDropdown.js

```diff
@@ -15,88 +15,50 @@ type Props = {
   visibleProjects: Array<Project>
 }
 
-export default class HomeProjectDropdown extends Component<Props> {
+type State = {
+  selectedOption: any
+}
+
+export default class HomeProjectDropdown extends Component<Props, State> {
+  state = {
+    selectedOption: ''
+  };
+
+  handleChange = (selectedOption: any) => {
+    this.setState(
+      {selectedOption},
+      () => console.log(`Option selected:`, this.state.selectedOption)
+    )
+  };
```

Comment:
@robertgregg3, sorry if there was confusion about the "routing to the project page" discussion. The behavior of `onChange` handler should remain the same as the previous selector. I.e., it should route to `{`/home/${project.id}`}`, and should not use the component state.